### PR TITLE
Reverts our sprites

### PR DESCRIPTION
**Pull request by ghost**
Ok, we're all just acting like retarded children now. Let's just learn to accept new things, k?

### CODE REVIEW

File: code/modules/reagents/reagent_dispenser.dm [hunk 1]
```diff
@@ -25,9 +25,7 @@
 
 /obj/structure/reagent_dispensers/Initialize()
 	create_reagents(tank_volume)
-	if(reagent_id)
-		reagents.add_reagent(reagent_id, tank_volume)
-	generate_reagent_icon()
```
Comment (GuyonBroadway) [inline code/modules/reagents/reagent_dispenser.dm:30]: this fucks over lean

Comment (ghost) [reply code/modules/reagents/reagent_dispenser.dm:30]: No it doesn't.

Comment (ghost) [reply code/modules/reagents/reagent_dispenser.dm:30]: This just reverts like, water tank / fuel tank / chem tank sprites. I didn't touch drinking glasses.

**Comment by dreamcrab** [comment]
so whats this do other than make you look like the retarded child

**Comment by ghost** [comment]
@dreamcrab make the sprites look better

**Comment by tortellinitony** [comment]
our sprites are new since we didn't have them yesterday, learn to accept them k?

**Comment by ghost** [comment]
@tortellinitony they are not new, as they were the default before the new sprites

**Comment by ghost** [comment]
-1

**Comment by ghost** [comment]
@zamolxius what do you have against good sprites

**Pull request closed without merging**